ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -236,6 +238,7 @@ public static void bootstrapOrUpgrade(String hardwareLayoutPath, String partitio
           clusterMapToHelixMapper.clusterName);
     }
     clusterMapToHelixMapper.updateClusterMapInHelix(startValidatingClusterManager);
+
```

Comment:
if u cud please minimize these newlines and formatting chagnes, then i can focus on the actual changes.